website/content/docs/release-notes/v0_14_0.mdx

```diff
@@ -199,6 +199,27 @@ description: |-
     </td>
   </tr>
 
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.14.0
+    <br /><br />
+    (Fixed in 0.13.5 and 0.14.1)
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Go CVE-2023-39326
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    The version of Go that was used in Boundary release 0.14.0 contained a CVE. The issue was fixed in Go version 1.21.5. Boundary was updated to use the new Go versions in release 0.14.1, and the issue is resolved.
+    <br /><br />
+    Note that CVE-2023-39326 was also fixed in version 0.13.5 of HCP Boundary and Boundary Enterprise.
+    <br /><br />
+    Learn more:&nbsp;
+    <a href="https://github.com/advisories/GHSA-9f76-wg39-x86h">A malicious HTTP sender can use chunk extensions</a>
+    <br /><br />
+    <a href="/boundary/tutorials/self-managed-deployment/upgrade-version">Upgrade to the latest version of Boundary</a>
+    </td>
+  </tr>
+
```

Comment:
At the moment, we note that 0.14.1 includes a bumped version of GO(1.21.5), however 0.14.1 was only bumped to what was available at the time, 1.21.3. Boundary 0.14.3 includes a bump to GO 1.21.5. I've Modified the following from the changelog for reference: 

**0.14.1** 
* deps: Bump Go version to v1.21.3; gRPC to v1.58.3; golang.org/x/net to
  v0.17.0. This is to address a security vulnerability in the HTTP stack where a
  malicious HTTP/2 client which rapidly creates requests and immediately resets
  them can cause excessive server resource consumption.
* Go CVE-2023-39325 & Go CVE-2023-39326

**0.14.3:** 
* deps: Bump Go version to v1.21.5. This is to address multiple security
  vulnerabilities, including a fix in net/http to limit
  chunked data overhead. See https://groups.google.com/g/golang-announce/c/iLGK3x6yuN
* CVE 2023-39322 & CVE 2022-45285

Reply:
Thank you for the clarification here! I think I updated this file accordingly, but the link the Google group conversation for the 0.14.3 fix was broken. So I included the links to the GitHub advisories for CVE-2023-39322 & CVE-2022-45285 instead.